docker/test-qemu.py

```diff
@@ -0,0 +1,121 @@
+subprocess.check_output(("tar -xzf "+ testfile_dir + "/iot-testfiles.intel-corei7-64.tar.gz  -C iottest/").split())
+
+try:
+    qemu_process = subprocess.Popen(qemu_args.split())
+    qemu_started = 1
+    # Wait some time for QEMU to boot and start trying to connect to it
+    sleep(30)
+    start_time = time()
+    while time() < (start_time + 120):
+        try:
+            subprocess.check_output("ssh -o UserKnownHostsFile=/dev/null " \
+                            "-o StrictHostKeyChecking=no " \
+                            "root@127.0.0.1 -p 2222 echo $?".split())
+            break
+        except:
+            sleep(1)
+
+    runtest_process = subprocess.Popen(runtest_args.split())
+    tests_started = 1
+    # Wait for test run to end
+    start_time = time()
+    while time() < (start_time + 120):
+        if not runtest_process.poll() is None:
+            subprocess.call("ssh -o UserKnownHostsFile=/dev/null " \
+                            "-o StrictHostKeyChecking=no " \
+                            "root@127.0.0.1 -p 2222 poweroff".split())
+            sleep(10)
+        if not qemu_process.poll() is None:
+            break
+        sleep(5)
+
+finally:
+    # Get list of log and test result files
+    logs = glob.glob("*.log")
+    tests = glob.glob("TEST-*")
+
+    # Count failed, passed and errors from test result files
+    errors_count = failed_count = skipped_count = tests_count = 0
+    for test in tests:
+        with open(test, "r") as f:
+            contents = f.read()
+            errors_count += int(re.findall('errors="([0-9]*)', contents)[0])
+            failed_count += int(re.findall('failures="([0-9]*)', contents)[0])
+            skipped_count += int(re.findall('skipped="([0-9]*)', contents)[0])
+            tests_count += int(re.findall('tests="([0-9]*)', contents)[0])
+
+    # Make results-summary-qemu.refkit-image-common.log file
+    passed_count = tests_count - errors_count - failed_count - skipped_count
+    total_count= passed_count + failed_count
+    image_name = os.path.basename(os.path.realpath(WORKSPACE + \
+    "/build/tmp-glibc/deploy/images/intel-corei7-64/refkit-image-common-intel-corei7-64.wic"))
+    with open("results-summary-qemu.refkit-image-common.log", "w") as f:
+        f.write("Image: "+image_name+"\n")
+        f.write("  Device: QEMU\n")
+        f.write("  Total:"+str(tests_count)+"  Pass:"+str(passed_count))
+        f.write("  Fail:"+str(failed_count)+"  Skip:"+str(skipped_count))
+        f.write("  Error:"+str(errors_count)+"\n")
+        if total_count > 0:
+            run_rate = 100*total_count/tests_count
+            pass_rate_of_total = 100*passed_count/tests_count
+            pass_rate_of_exec = 100*passed_count/total_count
+            f.write("  Run rate:" + str(run_rate) + "%")
+            f.write("  Pass rate of total:" + str(pass_rate_of_total) + "%")
+            f.write("  Pass rate of exec:" + str(pass_rate_of_exec) + "%\n")
+            f.write(67*"-" + "\n")
```

Comment:
thanks, but this feedback should go to #94, where I picked few commits from.
My intention in this PR is only to get KVM mode enabled, and to verify,
I re-used #94 commits to enable some qemu launching.
I think #94 is going under re-design anyway